src/app/util/attribute-storage.cpp

```diff
@@ -291,6 +292,31 @@ CHIP_ERROR emberAfSetDynamicEndpoint(uint16_t index, EndpointId id, const EmberA
         }
     }
 
+    for (uint8_t i = 0; ep && (i < ep->clusterCount); i++)
```

Comment:
If `ep` is null, we crashed back at the emberAfClusterCountForEndpointType call on line 279/280, so there's no point null-checking it here....

Reply:
Ack, will adjust.